lib/attack.go

```diff
@@ -204,6 +204,16 @@ func MaxBody(n int64) func(*Attacker) {
 	return func(a *Attacker) { a.maxBody = n }
 }
 
+// Client returns a functional option that allows you to BYO http.Client
```

Comment:
BYO is unnecessarily abbreviated and may lead to some users needing to lookup the term. Let's err on the side of explicitness here.